NEMO/views/users.py

```diff
@@ -481,6 +481,26 @@ def user_preferences(request):
     return render(request, "users/preferences.html", dictionary)
 
 
+@login_required
+@require_GET
+def view_user(request, user_id):
+    user = get_object_or_404(User, pk=user_id)
+
+    if request.user.id != user_id:
+        return HttpResponseBadRequest("Cannot view another user")
+
+    dictionary = {
+        "user": user,
+        "projects": Project.objects.filter(active=True, account__active=True),
+        "tool_qualifications": user.qualifications.all(),
+        "allow_profile_view": UserCustomization.get_bool("user_allow_profile_view"),
+        "groups": user.groups.all(),
+    }
+
+    if request.method == "GET":
+        return render(request, "users/view_user.html", dictionary)
```

Comment:
you already have the `@require_GET` so this check is not needed
```suggestion
    return render(request, "users/view_user.html", dictionary)
```